src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card-img">
+        <img
+          class="img-of-ware"
+          src="./images/imac.jpeg"
+          alt="iMac - 27 Retina image"
+        >
+      </div>
+
+      <div class="card__body">
+        <div class="card__title">
+          <h3 class="card__description">Моноблок APPLE A1419 iMac  27" Retina 5K (MNED2UA/A)</h3>
+          <p class="card__code">Код товара: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars">
```

Comment:
Переиспользуй код из layout_stars, тут еще должен быть класс-модификатор, который отвечает за количество активных звезд
```suggestion
          <div class="stars stars--4">
```